examples/src/main/java/io/dapr/examples/tracing/InvokeClient.java

```diff
@@ -52,18 +52,18 @@ public static void main(String[] args) throws Exception {
           InvokeMethodRequest request = builder
               .withBody(message)
               .withHttpExtension(HttpExtension.POST)
-              .withContext(getReactorContext()).build();
+              .build();
           client.invokeMethod(request, TypeRef.get(byte[].class))
               .map(r -> {
-                System.out.println(new String(r.getObject()));
+                System.out.println(new String(r));
                 return r;
               })
               .flatMap(r -> {
                 InvokeMethodRequest sleepRequest = new InvokeMethodRequestBuilder(SERVICE_APP_ID, "proxy_sleep")
                     .withHttpExtension(HttpExtension.POST)
-                    .withContext(r.getContext()).build();
+                    .build();
                 return client.invokeMethod(sleepRequest, TypeRef.get(Void.class));
-              }).block();
+              }).subscriberContext(getReactorContext()).block();
```

Comment:
Please, change the README.md file too to reflect this version.

Reply:
Updated